conbench/entities/history.py

```diff
@@ -56,6 +56,7 @@ def get_history(case_id, context_id, hardware_hash):
         .filter(
             BenchmarkResult.case_id == case_id,
             BenchmarkResult.context_id == context_id,
+            BenchmarkResult.mean.isnot(None),
```

Comment:
I'm find doing this now, but I wonder if we actually want to be more explicit and assert `BenchmarkResult.error == False`

Reply:
Changed this to `BenchmarkResult.error == None` since `BenchmarkResult.error` is JSON.